lib/components/primitive-components/Trace/Trace.ts

```diff
@@ -783,7 +785,53 @@ export class Trace
           {
             port: toPort,
             anchorPos: toAnchorPos,
-            schDisplayLabel: this.props.schDisplayLabel!,
+            schDisplayLabel: netLabelText!,
+            source_trace_id: this.source_trace_id!,
+          },
+          { db },
+        )
+      }
+      return
+    } else if (
+      lowercaseNetLabelText?.includes("vcc") ||
+      lowercaseNetLabelText?.includes("vdd")
```

Comment:
Making a note: This is fine for now, in the future we'll have net characteristics (technically we already do but it's quite old code) that will determine whether something is VCC/VDD or GND